src/components/DropDown/story.js

```diff
@@ -50,7 +50,7 @@ storiesOf('DropDown', module)
             margin: '0',
             padding: '0'
           }}>
-            <li>
+            <li className={styles.menuItem}>
```

Comment:
Without this, extra padding and margin will be applied to `<li>` inside DropDown, and that's not what we want 👍